### DISCUSSION TITLE
Terms Updates

### DISCUSSION THREAD

**Discussion by clementbiron** [OpenTermsArchive/terms-types#14] [Submitted]
### What are some documents containing these terms?

| Example | Service Name | URL | Title | Jurisdiction(s) |
|:----|:----|:----|:------|:-------------|
| 1 | X | [URL 🔗](https://help.twitter.com/en/rules-and-policies/update-privacy-policy) | Updates to our Terms of Service and Privacy Policy | 🌐 |
| 2 | Zora | [URL 🔗](https://support.zora.co/en/articles/1369217) | Summary of changes | 🌐 |
| 3 | Discord | [URL 🔗](https://discord.com/archive) | Terms of Service and Policy Archive | 🌐 |

### What could be some alternative names for this type?

Terms Updates

### Whose rights and duties are defined in these terms?

end user

### What is the object of commitment in these terms?

...

### Are there any relevant references for this type?

...

### Do you have any additional comments or information?

...

**Comment by afisher3578** [comment]
Zora - Summary of Changes can be added to this list: https://support.zora.co/en/articles/1369217

**Reply by clementbiron** [reply]
Added thank you!

**Comment by afisher3578** [comment]
https://discord.com/archive

**Reply by clementbiron** [reply]
Added, thank you!
